src/app/pages/choose-account/components/accounts.tsx

```diff
@@ -144,6 +145,7 @@ export const Accounts = memo(() => {
   const accounts = useAccounts();
   const { decodedAuthRequest } = useOnboardingState();
   const [selectedAccount, setSelectedAccount] = useState<number | null>(null);
+  useBackupOldWalletSalt();
```

Comment:
I start the backup from the account & the home page. It might not be ideal. Not sure what other options we have, since we need to wallet state.